azure-iot-device/azure/iot/device/common/pipeline/pipeline_stages_base.py

```diff
@@ -181,12 +182,15 @@ class PipelineRootStage(PipelineStage):
     :type on_disconnected_handler: Function
     """
 
-    def __init__(self):
+    def __init__(self, configObj=None):
```

Comment:
>configObj [](start = 23, length = 9)

I dont think python has camel case..it may work but generally we name our variables as underscored separations of words...

for example client id is `client_id` but not 'clientId`